builtin/v11/migration/system.go

```diff
@@ -0,0 +1,36 @@
+package migration
```

Comment:
Since we are now free away from specs actors versioning needs we should move migration infrastructure here and in top.go into a version independent file.  Then we can have migration specific setting of migrators.

When outer tree structure and system actor manifest state changes we'd have to abstract some of this behind interfaces or copy things like this.